research/engine/pegasus/pegasus/src/lib.rs

```diff
@@ -324,6 +327,7 @@ fn allocate_local_worker(conf: &Arc<JobConf>) -> Result<Option<WorkerIdIter>, Bu
 }
 
 use std::sync::Once;
+use std::net::SocketAddr;
```

Comment:
Move these `use` to the head of the file

Reply:
Done